### PR TITLE
AC Convert Test - LPD-32688 and LPD-32219 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
**Jira ticket:**

- https://liferay.atlassian.net/browse/LPD-32219
- https://liferay.atlassian.net/browse/LPD-32688

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:relevant

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:relevant

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:relevant

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:relevant

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

File: modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts [hunk 1]
```diff
@@ -111,6 +111,45 @@ export async function syncAnalyticsCloud({
...
+	organizationName,
+	page,
+	userGroupName,
+}: {
+	organizationName?: string;
+	page: Page;
+	userGroupName?: string;
+}) {
+	const selectContactsCollapsed = page
+		.locator('[aria-expanded="false"]')
```
Comment (marcosapmf) [inline modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts:224]: @ViniLopes20, this seems good, but do we have a more specific selector we can use? I think this one is too generic :sweat_smile: 
If we don't, it's ok then :)

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

File: modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts (hunk 1 above)
Comment (ViniLopes20) [reply modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts:224]: Hey @marcosapmf, In this screen we have two Collapsed panels and what differentiates them is only the Text and the value of `aria-expanded`. 

Should I put some locator to get a class that demonstrates that we are looking at these two panels and then see the `aria-expanded`?

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud-acceptance

File: modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts (hunk 1 above)
Comment (marcosapmf) [reply modules/test/playwright/tests/analytics-settings-web/utils/analyticsSettings.ts:224]: hmmm do we have any location in an outer context that lets clear what we are looking at?

**Comment by ViniLopes20** [comment]
ci:test:stop

**Comment by ViniLopes20** [comment]
ci:stop

**Comment by ViniLopes20** [comment]
Covered by: https://github.com/liferay-ac/liferay-portal/pull/1723

**Pull request closed without merging**